pxr/imaging/hio/glslfxConfig.cpp

```diff
@@ -617,7 +617,7 @@ HioGlslfxConfig::_GetAttributes(VtDictionary const & dict,
 
     const VtDictionary& attributesDict =
         attributes.UncheckedGet<VtDictionary>();
-    for (const std::pair<std::string, VtValue>& p : attributesDict) {
+    for (const std::pair<std::string, VtValue> p : attributesDict) {
```

Comment:
You should be able to avoid the copy by making `p` `const auto&` or `const std::pair<const std::string, VtValue>&`.